lib/dash/content_protection.js

```diff
@@ -366,16 +366,48 @@ shaka.dash.ContentProtection = class {
     ];
   }
 
+  /**
+   * Creates ClearKey initData from Default_KID value.
+   * @param {shaka.dash.ContentProtection.Element} element
+   * @param {!Set.<string>} keyIds
+   * @return {?Array.<shaka.extern.InitDataOverride>}
+   * @private
+   */
+  static getInitDataClearKey_(element, keyIds) {
+    if (keyIds.size == 0) {
+      return null;
+    }
+
+    const Uint8ArrayUtils = shaka.util.Uint8ArrayUtils;
+    const [first] = keyIds;
```

Comment:
It seems like you should probably use all the key IDs in constructing the PSSH, instead of just the first one.